src/lib/holocene/navigation/navigation-item.svelte

```diff
@@ -29,7 +32,8 @@
     aria-disabled={disabled}
     class:disabled
     tabindex={disabled ? -1 : 0}
-    class="mb-1 flex items-center whitespace-nowrap rounded-lg p-1 pl-2 text-sm font-medium hover:bg-white hover:text-black group-[.surface-primary]:hover:bg-black group-[.surface-primary]:hover:text-white group-[.surface-primary]:dark:hover:bg-white group-[.surface-primary]:dark:hover:text-black"
+    class:active={isActive($page.url.href)}
```

Comment:
Get a lot of errors for this lil guy

Reply:
Whenever you change namespace in namespace switcher it errors and crashes UI

Reply:
Added a [fix](https://github.com/temporalio/ui/pull/2324/commits/79b803a6370752e13cf6ac819584aa5d7cb3edb3). Looks like `isActive` was still `undefined` despite setting `() => false` as the default initial value 🤔